app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -92,6 +92,7 @@ module.exports = React.createClass
     annotationValues = @props.annotation.value
     unless annotationValues.length
       @props.annotation.value = @props.task.selects.map -> {value: null, option: false}
+      @syncAnnotations()
```

Comment:
It is, but I like your idea of separate PRs, I'll move this bug fix to Issue #3488 as well, or can open a different issue.